meta-lmp-bsp/recipes-security/optee/optee-os-fio-bsp.inc

```diff
@@ -91,11 +91,14 @@ EXTRA_OEMAKE:append:stm32mp15-eval = " \
 "
 
 # Additional Settings for SE05X
+EXTRA_OEMAKE:append = " \
+    ${@bb.utils.contains('MACHINE_FEATURES', 'se05x', 'CFG_PKCS11_TA_HEAP_SIZE=262144', 'CFG_PKCS11_TA_HEAP_SIZE=65536', d)} \
```

Comment:
I think it is probably a good idea to set with and without se050, as then we can have the same value set all over.

Reply:
As we discussed, better indeed to let it only when se050 is used, and for that you can move this config to 'meta-lmp-base/recipes-security/optee/optee-os-fio-se05x.inc', which is only included when se050 is enabled.

Reply:
The problem is that default value for now is 32KB. This is why I added it exactly here:
1. Increase to 64KB if se05x is not used
2. Set to 256KB is se05x is used


If I move this config meta-lmp-base/recipes-security/optee/optee-os-fio-se05x.inc, I'll still need this condition here, which will check for se05x

Reply:
@igoropaniuk will this work?
In `optee-os-fio-bsp.inc`:
```
PKCS11_TA_HEAP_SIZE ??= 65536
EXTRA_OEMAKE:append = " CFG_PKCS11_TA_HEAP_SIZE=${PKCS11_TA_HEAP_SIZE}"
```
In `optee-os-fio-se05x.inc`:
```
PKCS11_TA_HEAP_SIZE = 262144
```

Reply:
@MrCry0 thanks for suggestions, redone!

Reply:
@ricardosalveti comment addressed